.release-please-manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-  ".": "1.8.0",
-  "packages/api-explorer": "0.9.9",
-  "packages/code-editor": "0.1.2",
+  ".": "1.9.0",
+  "packages/api-explorer": "0.9.10",
+  "packages/code-editor": "0.1.3",
   "packages/extension-api-explorer": "21.8.0",
-  "packages/extension-sdk": "21.8.0",
-  "packages/extension-sdk-react": "21.8.0",
+  "packages/extension-sdk": "21.8.1",
+  "packages/extension-sdk-react": "21.8.1",
   "packages/hackathon": "21.8.0",
-  "packages/run-it": "0.9.9",
-  "packages/sdk": "21.8.0",
-  "packages/sdk-codegen": "21.0.17",
-  "packages/sdk-codegen-scripts": "21.0.18",
+  "packages/run-it": "0.9.10",
+  "packages/sdk": "21.8.1",
+  "packages/sdk-codegen": "21.0.18",
+  "packages/sdk-codegen-scripts": "21.0.19",
```

Comment:
Guess I made a change to scripts but not codegen. Do we want to keep these in sync?

Reply:
I'd rather not impose any forced version syncing that we don't have to. if you think there's a good reason then we can do it